Directory.Build.props

```diff
@@ -30,6 +30,13 @@
 		<Build_Windows>true</Build_Windows>
 	</PropertyGroup>
 
+	<ItemGroup>
+        <PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="4.7.0-2.final">
```

Comment:
Do we specifically need something from 4.7? If so, please add comment, otherwise drop back to latest stable (4.6)